content/en/introduction/what-is-kubesphere.md

```diff
@@ -6,38 +6,57 @@ description: 'What is KubeSphere'
-We decouple some main feature components and make them pluggable and optional to choose so that users can install a default KubeSphere with resource requirements down to 2 cores CPU and 4G memory. Meanwhile, there are great enhancements in application store, especially in application lifecycle management.
+  - **Network Security and Isolation**. Security policies are created based on the network of tenants. KubeSphere 3.0 supports the management of security policies of tenant network, which can be completely isolated from each other. Users themselves can also decide whether services are accessible externally. 
 
-It is worth mentioning that both DevOps and observability components have been improved significantly. For example, we add lots of new features including Binary-to-Image, dependency caching support in pipeline, branch switch support and Git logs output within DevOps component. We also bring upgrade, enhancements and bugfix in storage, authentication and security, as well as user experience improvements. See [Release Notes For 2.1.0](../../release/release-v210) for details.
+  - **Support Open Policy Agent**. Users can manage their security and authorization policies in a unified way with a general architecture.
+
+  - **Support OAuth 2.0**. Users can now easily integrate third-party applications with OAuth 2.0 protocol.
+
+- **Multilingual Support of Web Console**. KubeSphere is designed for users around the world at the very beginning. Thanks to our community members across the globe, KubeSphere 3.0 now supports five official languages for its web console: English, Simplified Chinese, Traditional Chinese, Spanish and Turkish. More languages are expected to be supported going forward.
+
+In addition to the above highlights, KubeSphere 3.0 also features other functionality upgrades. For more and detailed information, see Release Notes for 3.0.0.
 
 ## Open Source
 
-As we adopt open source model, development is taking in the open way and driven by KubeSphere community. KubeSphere is **100% open source** and available on [GitHub](https://github.com/kubesphere/) where you can find all source code, documents and discussions. It has been widely installed and used in development testing and production environments, and a large number of services are running smoothly in KubeSphere.
+As we adopt the open source model, development is proceeding in an open way and driven by KubeSphere community. KubeSphere is **100% open source** and available on [GitHub](https://github.com/kubesphere/) where you can find all the source code, documents and discussions. It has been widely installed and used in development, testing and production environments, and a large number of services are running smoothly in KubeSphere.
 
 ## Roadmap
 
 ### Express Edition -> KubeSphere 1.0.x -> KubeSphere 2.0.x -> KubeSphere 2.1.x -> KubeSphere 3.0.0
 
 ![Roadmap](https://pek3b.qingstor.com/kubesphere-docs/png/20190926000413.png)
 
-## Landscapes
+## Prospect and Aspiration
```

Comment:
Remove this paragraph please, I think the prospect is not proper to be included in the introduction.